datalake/src/main/java/com/sequenceiq/datalake/service/sdx/StorageValidationService.java

```diff
@@ -121,4 +192,12 @@ private Map<String, String> getUserMapping(String dataAccessRole, String rangerA
         }
         return userMapping;
     }
+
+    private Credential getCredential(DetailedEnvironmentResponse environment) {
+        return new Credential(environment.getCloudPlatform(),
+                environment.getCredential().getName(),
+                secretService.getByResponse(environment.getCredential().getAttributes()),
+                environment.getCredential().getCrn(),
+                environment.getCredential().getAccountId());
+    }
```

Comment:
This does not seem to be used.

Reply:
My original changes used it. Missed cleaning it up later.